server/verify-brackets.js

```diff
@@ -0,0 +1,18 @@
+const config = require('server/routes/config');
+
+const isReachable = require('is-reachable');
+var brambleServer = config.editorHOST;
+
+console.log("Checking for Bramble Server. " + brambleServer);
```

Comment:
In the spirit of Unix, let's only print in the case of an error.  You can drop this.